src/main/java/frc/robot/commands/AmpCommand.java

```diff
@@ -21,7 +20,8 @@ public AmpCommand(Shooter shooter) {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    shooter.startMotors(speed, Position.AMP);
+    shooter.setAngle(36);
```

Comment:
I know this is a magic value right now, but I'd still prefer it to be in a constant field or something with a TODO stating that it needs to be tuned. You will also want to specify once you tune it where on the field the robot needs to be for that angle to be correct.